### DISCUSSION TITLE
null authorizationCode when signing in with apple on Android

### DISCUSSION THREAD

**Discussion by pamafe1976** [firebase/flutterfire#12779] [Q&A]
Hi

I'm using this flutter code to retrieve credential for sign in with apple:

```
      final appleProvider = AppleAuthProvider()
        ..addScope('email')
        ..addScope('name');

      UserCredential credential = await FirebaseAuth.instance.signInWithProvider(appleProvider);
```

In IOS it returns a valid authorizationCode. However in Android the authorizationCode is always null.
I need the authorizationCode to be able to revokeTokenWithAuthorizationCode when the user deletes the account.
What am I doing wrong?

Regards

**Comment by zhouyuePi** [comment]
Were you able to figure it out? I am facing the same issue.
